### PR TITLE
added script tag to index head section

### PR DESCRIPTION
**Pull request by ArchitectFromMars**
added the <script> tag to the head of index.html

### CODE REVIEW

**Comment by terrifricker** [comment]
Great job, @ArchitectFromMars 

Question?  Do we want it to be at the end of body instead?

**Comment by ArchitectFromMars** [comment]
I considered that, given how it's a best practice to load js files after the page loads. However, given the naming convention "app.js" it reminded me of React.js where, I believe if I'm not mistaken, is actually the file that holds the entire app itself and so I thought it might need to be at the top. Thinking about it a second time here... I think moving it just above the body-end tag is the right place. Will update this in 15min or so!

**Comment by ArchitectFromMars** [comment]
alrighty. hope that works now.

**Comment by terrifricker** [comment]
Thanks @ArchitectFromMars.  I didn't actually know what to call the javascript file.  I usually use my project-name.js  We can change it to whatever we want once we know more about the project.  Or use script.js?  I didn't know that app.js was related to react.

**Review by terrifricker — APPROVED** [review]
@terrifricker approves

File: index.html [hunk 1]
```diff
@@ -7,6 +7,7 @@
         <link rel="stylesheet" href="styles.css">
     </head>
     <body>
-        
+        // code goes here
+        <script type="text/javascript" src="app.js"></script>
```
Comment (juancwu) [inline index.html:11]: Cool! Lets merge the code.

**Review by juancwu — APPROVED** [review]

**Pull request merged**